src/test/scala/edu/berkeley/ce/rockslicing/InputProcessingSpec.scala

```diff
@@ -0,0 +1,91 @@
+package edu.berkeley.ce.rockslicing
+
+import org.scalatest.FunSuite
+
+import scala.io.Source
+
+class InputProcessingSpec extends FunSuite {
+
+  test("Input file without \"%\" should produce error") {
```

Comment:
A test should also be added to check whether the global origin coordinates have been entered.